### DISCUSSION TITLE
Create new standard object Event

### DISCUSSION THREAD

**Discussion by FelixMalfait** [twentyhq/twenty#6984] [Ideas]
Unlike Salesforce our Event object will not represent a CalendarEvent (that will be another standard object).
Instead, it will be inspired by [Census](https://docs.getcensus.com/basics/data-models-and-entities/defining-source-data/events), [Segment](https://segment.com/docs/connections/spec/track/), [Posthog](https://posthog.com/docs/data/events).

**Fields**
- id
- createdAt
- happensAt
- type (standard such as pageview, object_created, object_modified, object_deleted ; or prefixed by _ for custom events)
- properties (json)
- personId, companyId...

**Comment by Bonapara** [comment]
Perhaps we could name it "Activity" instead of "events" to avoid confusion with calendar events.

**Comment by BOHEUS** [comment]
@FelixMalfait is it still relevant?
